README.md

```diff
@@ -343,6 +343,31 @@ You can enable the interval and set the interval time, along with your desired p
 
 ```
 
+## Write to file
+
+If you would like to write the json report to a file on disk, you can enable the write to file attribute.
+  Note:  If write to file is enabled, interval timing may not be enabled.  This is because when we write
```

Comment:
Could we just touch a file to change its timestamp? Interval handling should not depend on json reporter